.docker/base/start-agent.sh

```diff
@@ -1,22 +1,27 @@
 #!/bin/bash
+#Prepare everest-ci and everest-logs repositories; adding environment variable
+#PAT and agent values should be provided in the docker run command
+
+cd $MYHOME
+#Add environment variable for slack notifications
+export SLACK_FSTAR_WEBHOOK=https://hooks.slack.com/services/${PAT_SLACK}
+echo 'export SLACK_FSTAR_WEBHOOK="'"$SLACK_FSTAR_WEBHOOK"'"' >> ${MYHOME}/.profile
```

Comment:
I added the quoting based on Tahina suggestion to exclude special characters. I changed back to usual form as you stating since it works this way and we know the characters that are expected in this line.